src/pages/Page.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+
+import jsonData from '../data.json';
+import Question from '../components/Question';
+import Button from '../components/Button';
+import Loader from '../components/Loader';
+
+const Page = () => {
+  const [question, setQuestion] = useState([]);
+  const [step, setStep] = useState(0);
+
+  useEffect(() => {
+    setQuestion(jsonData);
+  }, []);
+
+  const handleNext = () => {
+    setStep(step + 1);
+  };
+
+  const handlePrev = () => {
+    setStep(step - 1);
+  };
+
+  return (
+    <div>
+      {question ? (
+        <div className="flex flex-col justify-center items-center w-full m-auto p-14 bg-gray-400 h-screen">
+          {question[step] !== undefined && (
```

Comment:
what is it null ? or false ? or other falsy value ?
Don't use && better ternary operator

Reply:
If questions is null I have a loader.